serverless/knative_eventing/serverless-kafka.adoc

```diff
@@ -7,6 +7,9 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 You can use the `KafkaChannel` channel type and `KafkaSource` event source with {ServerlessProductName}.
+
+Using Kafka components in your event-driven architecture provides _at least once_ guarantees for event delivery. This means that operations are retried until a return code value is received. However, while this makes your application more resilient to lost events, it may result in duplicate events being sent.
```

Comment:
Apologies if I'm missing something.

Should we explicitly specify that we retry if it is specified to do so (eg `spec.delivery.retry > 0` for `KafkaChannel`)?

On the other hand, in `KafkaSource` we retry a limited number of times and there is no way to change that number, so I'm not sure if we can call it `at least once` when it's actually `at least once "modulo" retries` .

I'd love @matzew and @aliok opinion on this.

Reply:
> .... operations are retried until a return code value is received.

perhaps to ---> 

```text
.... operations are retried until a return code value is received, in case there is a retry configuration on the Delivery Spec
```

Reply:
Regarding _at least once_ 

How about:

> Using Kafka components in your event-driven architecture provides _at least once_ guarantees for event delivery. For the Kafak source there is a fixed number of build-in retries for event delivery, while for the Kafka channel retry is only performed, if that is configured on the Delivery Spec. However, while this makes your application more resilient to lost events, it may result in duplicate events being sent.

I think we can still call that "at least once" ... (rather than at-least-once-modulo-reties :sweat_smile: )